cmd/ocm-backplane/login/login.go

```diff
@@ -121,16 +121,56 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 		"ID":   clusterID,
 		"Name": clusterName}).Infoln("Target cluster")
 
+	// validate kubeconfig save path when login into multi clusters
+	if args.kubeConfigPath != "" {
+		if !args.multiCluster {
+			return fmt.Errorf("can't save the kube config into a specific location if multi-cluster is not enabled. Please specify --multi flag")
+		}
+		if _, err := os.Stat(args.kubeConfigPath); errors.Is(err, os.ErrNotExist) {
+			return fmt.Errorf("kube config save path is not exist")
+		}
+	}
+
+	// Get Backplane URL
+	bpURL := globalOpts.BackplaneURL
+	if bpURL == "" {
+		bpURL = bpConfig.URL
+	}
+
+	if bpURL == "" {
+		return errors.New("empty backplane url - check your backplane-cli configuration")
+	}
+
+	var env string
+	if strings.Contains(bpURL, "integration") {
+		env = "integration"
+	} else if strings.Contains(bpURL, "stage") {
+		env = "staging"
+	} else {
+		env = "production"
+	}
+
```

Comment:
This may expose all available RH env's to the public. You may able to get the env with `ocm get /api/clusters_mgmt/v1/environment`  API or wait for [OSD-19188](https://issues.redhat.com/browse/OSD-19188) :)